tests/lib/acpi/src/main.c

```diff
@@ -8,6 +8,22 @@
 #include <zephyr/kernel.h>
 #include <zephyr/acpi/acpi.h>
 
+#if (ACPI_DT_HAS_HID(DT_NODELABEL(pcie0)))
+#define PCIE_HID ACPI_DT_HID(DT_NODELABEL(pcie0))
+#define PCIE_UID ACPI_DT_UID(DT_NODELABEL(pcie0))
```

Comment:
Here it would make sense to use `DT_PROP_OR(..., ..., NULL)`, so that you don't need to have the `#else` branch with some arbitrary looking "INVALID" strings (even if you did have them NULL would be better)